AutoCal/Controller/SchedulerVC.swift

```diff
@@ -0,0 +1,113 @@
+//
+//  SchedulerViewController.swift
+//  AutoCal
+//
+//  Created by Ilan Harari on 4/22/20.
+//  Copyright © 2020 Ilan Harari. All rights reserved.
+//
+
+import UIKit
+import EventKit
+import EventKitUI
+class SchedulerVC: UIViewController {
+    var eventStore = EKEventStore()
+    var reminder: EKReminder!
+    
+    @IBOutlet weak var expensePicker: UIDatePicker!
+    @IBOutlet weak var PlannerView: UILabel!
+    
+    
+    //MARK: ViewDidLoad()
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        PlannerView.text = reminder.title
+        
+        switch EKEventStore .authorizationStatus(for: .event) {
+               
+           case .notDetermined:
+               print("access to event store undetermined. Trying again now...")
+               eventStore.requestAccess(to: .event, completion: {(granted: Bool, error: Error?) -> Void in
+                   if granted {
+                      print("access granted")
+                    //call scheduleEvent()
+                   } else {
+                       self.accessAlert()
+                   }})
+           case .restricted:
+               print("Access restricted.")
```

Comment:
It's not part of this story, so feel free to ignore for now: This should be a modal window error like in the reminders view